Doppler.HtmlEditorApi/Infrastructure/IDummyRepository.cs

```diff
@@ -0,0 +1,12 @@
+using Doppler.HtmlEditorApi.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Doppler.HtmlEditorApi.Infrastructure
+{
+    public interface IDummyRepository
```

Comment:
What do you think about also adding the methods to store the data?

Reply:
Fixed